daterangepicker.js

```diff
@@ -811,6 +815,16 @@
                     if (this.endDate != null && calendar[row][col] > this.startDate && calendar[row][col] < this.endDate)
                         classes.push('in-range');
 
+                    // Set custom classes configured
+                    var cellClasses;
+                    if (typeof this.dateCellClasses === 'function')
+                        cellClasses = this.dateCellClasses(calendar[row][col]);
+                    else 
+                        cellClasses = this.dateCellClasses;
+
+                    if ($.isArray(cellClasses))
```

Comment:
I used array because classes are arrays in the caller function. Don't really have a preference personally. I guess its up to the owner of the repo